libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -50,6 +53,19 @@ class LnurlPayInfoJob(
         try {
             request = Json.decodeFromString(LnurlInfoRequest.serializer(), payload)
             val nodeState = breezSDK.nodeInfo()
+            val ofpResp =
+                breezSDK.openChannelFee(OpenChannelFeeRequest(amountMsat = null))
+            val feeLimitMsats: ULong = config.autoChannelSetupFeeLimitMsats
+            val maxReceivableMsatsThatFallsWithinFeeLimits = minOf(
+                nodeState.maxReceivableMsat,
+                feeLimitMsats / (ofpResp.feeParams.proportional / 1000000u) / 100u,
```

Comment:
I still think this is wrong 🙂

feeLimitMsats: 10_000
proportional: 4_000
10_000 / (4_000 / 1_000_000) / 100 = 25_000

It should be 2_500_000

```suggestion
                feeLimitMsats * 1000000u / ofpResp.feeParams.proportional,
```
or
```suggestion
                feeLimitMsats / (ofpResp.feeParams.proportional / 1000000u),
```

Reply:
Using the second suggestion as it reads
`feeLimitMsats / proportional%`